app/src/main/res/drawable/ic_speaker_off.xml

```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There are `camera_off` and `camera_on` but only `speaker_off` - don't we need also `speaker_on`? 😄

Reply:
Right, the icon is not ready yet in figma.. I will add it later once it's available